### ISSUE TITLE
A rule to enforce `.dispatch(action())`

### ISSUE THREAD

**Issue by rafaelss95** [timdeschryver/eslint-plugin-ngrx#238]
## Information about the new rule
As mentioned in https://github.com/timdeschryver/eslint-plugin-ngrx/pull/211#discussion_r712270311, we could create a rule for this.

Do you have any name in mind?

## A snippet of incorrect code
```ts
this.store$.dispatch({ type: 'myAction' });
this.store$.dispatch(new MyAction());

```

## A snippet of correct code
```ts
this.store$.dispatch(action());
```
## I would be willing to submit a PR for the docs :heart:

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by timdeschryver** [comment]
What about `use-actions-in-dispatch` ?

**Comment by rafaelss95** [comment]
My only concern to this name is that `this.store$.dispatch(new MyAction())` would be flagged as well.. so imagine users asking theirselves: "wait, isn't `new MyAction()` an action? Is this a incorrect report?" 🤔

Maybe something like `use-action-creator-in-dispatch`?

**Comment by timdeschryver** [comment]
Works for me 👍
